lib/client/doc.js

```diff
@@ -908,3 +1022,281 @@ function callEach(callbacks, err) {
   }
   return called;
 }
+
+// *** Presence
+
+Doc.prototype.submitPresence = function (data, callback) {
```

Comment:
All of these methods could be definde only `if(ShareDB constructor presence flag)`